From f4dd85254e6edf43b303e923329e1126ac0341ef Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Tue, 2 Feb 2016 21:25:42 +0200
Subject: [PATCH 0775/1240] dts: a70x0: Add dev-info.dts for 7040-db-dop dts

  	include armada-70x0-dev-info.dtsi in armada-7040-rz-db_dop.dts
	To enable mppbus feature

Change-Id: I672230e6b3348a528652fa880dda27b8364cb731
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27158
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 arch/arm/dts/armada-7040-rz-db_dop.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/dts/armada-7040-rz-db_dop.dts b/arch/arm/dts/armada-7040-rz-db_dop.dts
index d14c533..895e7d6 100644
--- a/arch/arm/dts/armada-7040-rz-db_dop.dts
+++ b/arch/arm/dts/armada-7040-rz-db_dop.dts
@@ -19,6 +19,7 @@
 /dts-v1/;
 
 #include "armada-70x0.dtsi" /* include SoC device tree */
+#include "armada-70x0-dev-info.dtsi" /* include Marvell specific info (S@R, MPP cmd) */
 
 / {
 	model = "DB-ARMADA-7040R-Z1 (DOP)";
-- 
1.9.1

